src/services/role/index.js

```diff
@@ -40,11 +40,30 @@ module.exports = function setup() {
 
 	definePermissions(
 		'ADMIN_TOGGLE_STUDENT_VISIBILITY',
-		ROLES.TEACHER,
+		[ROLES.TEACHER],
 		PERMISSIONS.STUDENT_LIST,
 		// TODO: importent add this again, after school permission service can handle multiple permissions
 		// PERMISSIONS.STUDENT_EDIT,
 		// PERMISSIONS.STUDENT_CREATE,
 		// PERMISSIONS.STUDENT_DELETE,
 	);
+
+	definePermissions(
+		'LERNSTORE_MODE',
```

Comment:
This will not work and take no effect, as LERNSTORE_MODE is not matching the if statements in definePermissions as it consists other values, defined in https://github.com/schul-cloud/schulcloud-server/blob/a0a8128ad8cc2e503a73585b192d79750c93474e/config/default.schema.json#L328